Lib/ImageND/imagend.pd

```diff
@@ -617,11 +617,11 @@ use PDL::Options;
 sub PDL::convolveND {
```

Comment:
Do you think other variables in PDL::convolveND should be changed, too?  e.g., `$a0`->`$x0` and `$adims`->`$xdims` and `$aa`->`$xx`.

Reply:
I thought they would be fine, since it's only literally `$a` et al that have the problem.